src/components/breadcrumbs/breadcrumbs.types.ts

```diff
@@ -0,0 +1,31 @@
+import { Link } from '@limetech/lime-elements';
+
+export interface BreadcrumbsItem {
+    /**
+     * label displayed on the step.
+     */
+    text: string;
+
+    /**
+     * Icon of the step.
+     */
+    icon?: string;
```

Comment:
we may wanna wait for this new icon interface: https://github.com/Lundalogik/lime-elements/pull/2489 to be merged, so that we can use it here instead?